Vagrantfile

```diff
@@ -0,0 +1,26 @@
+Vagrant.configure(2) do |config|
+    config.vm.box = "fedora/25-cloud-base"
+    config.vm.network "forwarded_port", guest: 9090, host: 9090
+
+    config.vm.synced_folder ".", "/vagrant", disabled: true
+    config.vm.synced_folder "dist/", "/usr/local/share/cockpit/" + File.basename(Dir.pwd)
```

Comment:
As discussed on irc, I propose just creating this directory and printing a brief warning along the lines of `Distribution directory is currently empty, please call make`